ckanext/ckanext-apicatalog_scheming/ckanext/apicatalog_scheming/schemas/organization.json

```diff
@@ -129,6 +129,12 @@
       "validators": "ignore_not_sysadmin keep_old_value_if_missing default_value()",
       "output_validators": "scheming_load_json"
     },
+    {
+      "field_name": "metadata_updated_from_xroad_timestamp",
+      "validators": "ignore_not_sysadmin keep_old_value_if_missing default_value()",
+      "form_snippet": null,
```

Comment:
Should this be hidden.html? what happens if organization is updated from the form, does the timestamp disappear.

Reply:
the validators (`keep_old_value_if_missing specifically`)should take care of that